akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -174,7 +197,8 @@ object MaxThroughputSpec extends MultiNodeConfig {
       case EndResult(totalReceived) ⇒
         val took = NANOSECONDS.toMillis(System.nanoTime - startTime)
         val throughput = (totalReceived * 1000.0 / took)
-        println(
+
+        reporter.reportResults(
           s"=== MaxThroughput ${self.path.name}: " +
             f"throughput ${throughput * testSettings.senderReceiverPairs}%,.0f msg/s, " +
             f"${throughput * payloadSize * testSettings.senderReceiverPairs}%,.0f bytes/s (payload), " +
```

Comment:
ok, that would require one counter per sender in the Receiver? might add to much overhead.

just report it as -1 then, if targets more than one